osconfig_tests/test_suites/package_management/package_management.go

```diff
@@ -0,0 +1,187 @@
+//  Copyright 2018 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package package_management
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"regexp"
+	"sync"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/junitxml"
+	"github.com/kylelemons/godebug/pretty"
+	api "google.golang.org/api/compute/v1"
+
+	osconfigpb "github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/google-osconfig-agent/_internal/gapi-cloud-osconfig-go/google.golang.org/genproto/googleapis/cloud/osconfig/v1alpha1"
+
+	osconfig_server "github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/osconfig_server"
+)
+
+const testSuiteName = "PackageManagementTests"
+
+// TODO: Should these be configurable via flags?
+const testProject = "compute-image-test-pool-001"
+const testZone = "us-central1-c"
+
+var dump = &pretty.Config{IncludeUnexported: true}
+
+// TODO: Move to the new combined osconfig package, also make this easily available to other tests.
+var installOsConfigAgentDeb = `echo 'deb http://packages.cloud.google.com/apt google-osconfig-agent-stretch-unstable main' >> /etc/apt/sources.list
+curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+apt-get update
+apt-get install -y google-osconfig-agent
+echo 'osconfig agent install done'`
+var installOsConfigAgentGooGet = `c:\programdata\googet\googet.exe -noconfirm install -sources https://packages.cloud.google.com/yuck/repos/google-osconfig-agent-unstable google-osconfig-agent
```

Comment:
minor: consider a line break between these vars to make it a little easier to read.

Reply:
done